src/v2/ion/i18nTransformer.js

```diff
@@ -83,15 +97,22 @@ const I18N_OVERRIDE_MAPPINGS = {
   'challenge-authenticator.phone.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
   'challenge-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
   'challenge-authenticator.app.credentials.totp': 'oie.okta_verify.totp.enterCodeText',
+
+  'enroll-profile.userProfile.lastName': 'oie.user.profile.lastname',
+  'enroll-profile.userProfile.firstName': 'oie.user.profile.firstname',
+  'enroll-profile.userProfile.email': 'oie.user.profile.primary.email',
+
+  'idx.session.expired' : 'oie.idx.session.expired',
+
 };
 
 const getI18nKey = (i18nPath) => {
   let i18nKey;
 
-  // we can add mapping to `I18N_OVERRIDE_MAPPINGS` for all
-  // security question. It's just a bit tedious hence use following shortcut.
-  if (i18nPath.indexOf(SECURITY_QUESTION_PREFIX) === 0) {
-    const securityQuestionValue = i18nPath.replace(SECURITY_QUESTION_PREFIX, '');
+  // get security question key
+  if (i18nPath.indexOf(SECURITY_QUESTION_PREFIXES[0]) === 0 ||
+      i18nPath.indexOf(SECURITY_QUESTION_PREFIXES[1]) === 0) {
+    const securityQuestionValue = i18nPath.substr(i18nPath.lastIndexOf('.') + 1);
```

Comment:
would you be able to preserver the previous logic
`i18nPath.replace(SECURITY_QUESTION_PREFIX, '');`

```
SECURITY_QUESTION_PREFIXES.forEach(x => {
...
});
```